Tests/MultiTargetSpec.swift

```diff
@@ -54,10 +54,10 @@ class MultiTargetSpec: QuickSpec {
                 expect(String(describing: target.task)).to(beginWith("requestParameters")) // Hack to avoid implementing Equatable for Task
             }
 
-            it("uses correct sample data") {
-                let expectedData = "sample data".data(using: .utf8)!
-                expect(target.sampleData).to(equal(expectedData))
-            }
+//            it("uses correct sample data") {
```

Comment:
If it's commented out, just remove it :)

Reply:
Ah, I commented it out temporarily. I should've added a TODO. I don't think I should be reducing test coverage, but rather fix whatever tests I break 😅

Reply:
Adding a TODO is always good :) But if your PR is WIP and you remember to fix this later than its fine :)